### DISCUSSION TITLE
Hue Gradient via API

### DISCUSSION THREAD

**Discussion by p3tAbyte** [hyperion-project/hyperion.ng#1346] [Ideas]
Hi all,

Since a couple of days one can control the 7 different zones of the Hue Gradient LEDStrip via the app.
(https://www.reddit.com/r/Hue/comments/po9fay/hue_gradient_lightstrip_up_to_3_zones_can_be/)
(one can assign 3 base colors and the app will form a gradient between those colors.)

Within the API it's possible to address the 7 different zones. At the moment HyperionNG only supports
a couple of placements for Hue. Could this be something to fix up?

Thanks in advance

**Comment by Paulchen-Panther** [comment]
Please open a [Feature Request](https://github.com/hyperion-project/hyperion.ng/issues/new?assignees=&labels=feature+request&template=feature_request.md). Thank you.
